library/ssl_tls.c

```diff
@@ -3100,6 +3100,36 @@ static int ssl_handshake_init( mbedtls_ssl_context *ssl )
     }
 #endif
 
+#if defined(MBEDTLS_ECP_C)
+#if !defined(MBEDTLS_DEPRECATED_REMOVED)
+    /* Heap allocate and translate curve_list from internal to IANA group ids */
+    #define HEAP_MAX 66553
+    if ( ssl->conf->curve_list ) {
```

Comment:
New line for `{`

Reply:
We usually spell out non-zero checks, so add `!= NULL` here.

Reply:
Amended